src/components/MeasurementOption/MeasurementOption.js

```diff
@@ -155,9 +162,14 @@ class MeasurementOption extends React.Component {
       { value: 0.0001, name: '0.0001' },
     ];
 
+    if (hideScaleInputContainer && hidePrecisionInputContainer) {
+      return null;
```

Comment:
added this as we want a DOM element with height of 0 else it appears in DOM and looks weird